src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameSession.cs

```diff
@@ -473,7 +473,7 @@ private void UpdateConflictResolutionTask()
             var asyncToken = _asyncListener.BeginAsyncOperation(nameof(UpdateConflictResolutionTask));
 
             _conflictResolutionTask = _allRenameLocationsTask.SafeContinueWithFromAsync(
-               t => t.Result.GetReplacementsAsync(replacementText, optionSet, cancellationToken),
+               async t => await (await t.ConfigureAwait(false)).GetReplacementsAsync(replacementText, optionSet, cancellationToken).ConfigureAwait(false),
```

Comment:
Huh? This is a continuation, t.Result is always provided.

Reply:
(to be changed to a pragma disable)